lib/cmd/common/admin-storeitems.js

```diff
@@ -185,11 +185,11 @@ function update(guid, params, cb) {
 }
 
 function addIcon(guid, iconFile, cb) {
-  fhreq.uploadFile("/box/srv/1.1/admin/storeitem/uploadbinary", iconFile, {"type":"icon", "guid":guid}, "application/octet-stream" , cb);
+  fhreq.uploadFile("/box/srv/1.1/admin/storeitem/uploadbinary", iconFile, {"type":"icon", "guid":guid}, cb);
```

Comment:
Mmm, I get where you coming from. But I do think sometimes the content type needs to be explicitly set and now with this change it is not easy.

I think it's better to keep the option but allow it to be set to null. It is less change to be made in this PR, and also it will be useful for certain functions.